presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/TopNMatcher.java

```diff
@@ -61,18 +57,12 @@ public MatchResult detailMatches(PlanNode node, StatsProvider stats, Session ses
             return NO_MATCH;
         }
 
-        List<Symbol> expectedOrderBy = orderBySymbols.stream()
-                .map(alias -> alias.toSymbol(symbolAliases))
-                .collect(toImmutableList());
-
-        if (!topNNode.getOrderBy().equals(expectedOrderBy)) {
-            return NO_MATCH;
-        }
-
-        Map<Symbol, SortOrder> expectedOrderings = Maps.toMap(expectedOrderBy, Functions.constant(SortOrder.ASC_NULLS_FIRST));
```

Comment:
I don't understand this change. It looks surprising to me, given the commit is all about replacing two fields in `TopNNode` with single equivalent composite.

Reply:
As you can see old code implicitly assumed the same ordering for all columns which didn't allow to completely test different sorting cases. I could move this changes to separate commit, but I wonder if it's worth it.